### PR TITLE
Disable PostgreSQL logging_collector on Windows

### PR DESCRIPTION
**Pull request by Eprince-hub**
Closes https://github.com/upleveled/system-setup/issues/81

Turn off the `logging_collector` for Postgres to enable the log messages to display in the terminal on Windows

This applies only to Windows because:

- macOS students have not reported issues
- on Linux, PostgreSQL currently runs in the background (no foreground logging)
- some Windows students have reported problems similar to https://github.com/upleveled/system-setup/issues/81  

We decided to use `perl` similar to the solution we used in https://github.com/upleveled/system-setup/pull/80

Before command
Screenshot 2024-07-22 134907

After command
Screenshot 2024-07-22 135103

### CODE REVIEW

File: linux.md [hunk 1]
```diff
@@ -199,6 +199,7 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
     sudo apt update
     sudo apt install postgresql-16
     echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'`
+    perl -i -pe "s/^logging_collector = on/logging_collector = off/" "$PGDATA/postgresql.conf"
```
Comment (Eprince-hub) [inline linux.md:202]: Use perl to set the `logging_collector = off` to enable psql message logging in terminal.
 `PGDATA` variable will be available once https://github.com/upleveled/system-setup/pull/80 is merged

Comment (karlhorky) [reply linux.md:202]: we probably don't need this for Linux or macOS

Comment (Eprince-hub) [reply linux.md]: Remove here
faf6debff142f9ef66ed3629c5f11d8f12bbd9fc

File: windows.md [hunk 2]
```diff
@@ -256,6 +256,7 @@ With those compatibility things out of the way, you're ready to start the system
     echo "export PATH=\$PATH:\"/c/Program Files/PostgreSQL/16/bin\"" >> "$USERPROFILE/.bash_profile"
     echo "export PGDATA=\"/c/Program Files/PostgreSQL/16/data\"" >> "$USERPROFILE/.bash_profile"
     echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> "$USERPROFILE/.bash_profile"
+    perl -i -pe "s/^logging_collector = on/logging_collector = off/" "$PGDATA/postgresql.conf"
     perl -i -pe 's/^[#\s]*(lc_messages|lc_time)\s*=.+$/\1 = '\''en_US.UTF-8'\''/' "$PGDATA/postgresql.conf"
```
Comment (karlhorky) [inline windows.md:260]:
```suggestion
    perl -i -pe 's/^[#\s]*(lc_messages|lc_time)\s*=.+$/\1 = '\''en_US.UTF-8'\''/' "$PGDATA/postgresql.conf"
    perl -i -pe "s/^logging_collector = on/logging_collector = off/" "$PGDATA/postgresql.conf"
```

**Review by karlhorky — APPROVED** [review]

File: windows.md (hunk 2 above)
Comment (Eprince-hub) [reply windows.md]: Missed this suggestion so needed to manually add it

c7bbfa15444e5af1437589c2131a95315086c50c

**Review by karlhorky — APPROVED** [review]
LGTM, thanks!

**Comment by karlhorky** [comment]
Looks like we should replace the 2 screenshots that we have in the Windows guide with a single new text block (for maintainability), which doesn't contain the message about the logs being redirected:

Screenshot 2024-08-02 at 11 28 24

Opened an issue:

- https://github.com/upleveled/system-setup/issues/92

**Pull request merged**